src/CarbonSystemSolvers.jl

```diff
@@ -1,27 +1,27 @@
-module CarbonSystemSolver
+module CarbonSystemSolvers
 
-module CarbonSystemApprox
-export CarbonSolverApprox
+"""
+CarbonSolverApprox solves a cubic equation in terms of [H⁺]; 
+Not for serious use, but as a placeholder and for testing purposes
+"""
+module CarbonSolverApprox
```

Comment:
did you mean to have a module nested inside another in this file?

Reply:
Yes, in practice, a user will only need to use `CarbonSolverApprox`, `CarbonSolverFollows` (*new*), or `CarbonSolverMunhoven` (*tbd*). I'm still exploring a bit, but I'm leaning towards having the struct `CarbonSystem` and common functions to solve for pCO2, CO2aq, HCO3, and CO3 in the top-level module and then have the solver components separated off in sub-modules.